force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,8 +74,22 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if (num1 > num2 || num1 < num2) {
+            largestNum = Math.max(num1, num2);
+        }
+        else if (largestNum == num1) {
+            largestNum = num1;
+        }
+        else if (largestNum == num2) {
+            largestNum = num2;
+        }
+        else if (num1 == num2) {
+            largestNum = num1;
+        }
+        else if (num1 == null || num2 == null) {
+            largestNum = null;
```

Comment:
For practice try using Math.max. It wasn't talked about but I think you get figure it out.